db/index.js

```diff
@@ -0,0 +1,50 @@
+//https://mongoosejs.com/docs/index.html
+const set = require('./complete.js');
+
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost/photos', {useNewUrlParser: true});
+// const promise = require('bluebird');
+
+const db = mongoose.connection;
+db.on('error', console.error.bind(console, 'connection error:'));
```

Comment:
Please provide a function as the second input to `.on()`. You are providing a _function invocation_.